content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को बाहरी बनाना"
+content_type: tutorial
+weight: 10
+---
+
+<!-- overview -->
+
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों इक्स्टर्नलाइज़ बनाना है।
+विशेष रूप से, आप सीखेंगे कि एनवायरनमेंट वेरिएबल सेट करने के लिए कुबेरनेट्स कॉन्फिगमैप्स और सीक्रेट्स का उपयोग कैसे करें और फिर माइक्रोप्रोफाइल कॉन्फिग का उपयोग करके उनका उपभोग करें।
+
+
+## {{% heading "prerequisites" %}}
+
+### कुबेरनेट्स कॉन्फ़िगमैप्स और सीक्रेट बनाना
```

Comment:
Consider using English text for the link to prevent the presence of non-English characters in the URL of the section.

```suggestion
### कुबेरनेट्स कॉन्फ़िगमैप्स और सीक्रेट बनाना {#creating-kubernetes-configmaps-secrets}
```